monitoring/README.md

```diff
@@ -3,44 +3,101 @@
 <img src="images/ks-monitoring-setup.png" width="80%" height="80%" title="kubestellar-monitoring-setup">
 
 
+### Requirement
+To follow the instructions here, you need to have release `v0.2.0-rc12` or higher for the status-addon controller and agent installed in your enviroment. If you're using an older release, please follow these instructions to update your environment: 
+
+1.  Remove the status-addon-controller and status-addon-agent from your current environment:
+
+```bash
+helm --kube-context its1 -n open-cluster-management delete status-addon
+k --context its1 -n cluster1 delete manifestwork addon-addon-status-deploy-0
```

Comment:
Please write out `kubectl` --- aliasing it to `k` is not something that we have told users that they have to do.

Reply:
Fixed!